Feature/suscripciones panel (#2563)
mercadopago/devsite-docs: guides/subscription-plans/create-subscription-plan.pt.md

# Criar assinatura

Uma assinatura é uma ferramenta online que permite receber pagamentos de forma automática e recorrente. Ao criar um plano de assinatura, você define seu nome, preço e com que frequência cobrará seus clientes. Em seguida, você poderá compartilhá-lo por WhatsApp ou qualquer outra rede social.

Siga os passos abaixo para criar um plano de assinatura e começar a cobrar de forma recorrente, fácil e sem integração.

> ## Criar um Plano de Assinatura

Reviewer: ```suggestion
## Criar um plano de assinatura
```

* **Data de Cobrança:** escolha o dia do mês em que a assinatura será cobrada. _Esta opção está disponível apenas para planos com frequência de cobrança mensal_.
* **Cobrar valor proporcional na data de adesão:** Se esta opção estiver ativada, os clientes pagarão um valor proporcional ao se inscreverem pelos dias restantes até a primeira cobrança mensal. Se desativada, o valor adicional não será cobrado.
* **Código de Referência:** Permite adicionar um código de referência para ajudá-lo a identificar o plano, caso você tenha vários planos semelhantes ativos.
* **Para onde você gostaria de redirecionar seu cliente?:** Você pode adicionar uma URL para a qual o cliente será redirecionado uma vez que a assinatura for concluída, como por exemplo seu site.

Depois de preencher todos os campos, clique em **Criar e Compartilhar** para finalizar.

## Compartilhar um plano de assinatura

> Uma vez criado o plano de assinatura, você poderá compartilhar facilmente o link do seu plano. Para isso, clique em **Copiar Link** para colá-lo onde preferir ou selecione uma das opções oferecidas para enviá-lo diretamente por WhatsApp, Facebook, X ou e-mail.

Author: ```suggestion
Uma vez criado o plano de assinatura, você poderá compartilhar facilmente o link do seu plano. Para isso, clique em **Copiar Link** para colá-lo onde preferir ou selecione uma das opções oferecidas para enviá-lo diretamente por WhatsApp, Facebook, X ou e-mail.

```

> Você também pode colar um botão em seu site. Para isso, clique na opção **Ou cole um botão em seu site**. Um código HTML será exibido pronto para copiar e colar em sua página.

Author: ```suggestion
Você também pode colar um botão em seu site. Para isso, clique na opção **Ou cole um botão em seu site**. Um código HTML será exibido pronto para copiar e colar em sua página.

```

* **Preço:** defina o valor que as pessoas que se inscreverem pagarão. Além disso, você encontrará a caixa **Prefiro que o cliente defina**, que permite que os assinantes definam o valor que desejam pagar pela assinatura e, consequentemente, não permite definir um valor fixo para o plano de assinatura. Se você preferir esse modelo, clique na caixa para selecioná-la.
* **Frequência de Cobrança:** selecione com que frequência você deseja que seu plano de assinatura seja cobrado aos seus clientes. A frequência pode ser semanal, mensal, anual, quinzenal, bimestral, trimestral, quadrimestral ou semestral.

> WARNING
> 
> Importante
> 
> Ao criar uma assinatura, o nome da sua loja e seu e-mail serão compartilhados com o assinante.

### Personalizar plano de assinatura

Você pode personalizar ainda mais seu plano clicando em **Mais Opções**. Saiba como preencher cada campo:

* **Teste grátis:** permite oferecer um período gratuito aos assinantes antes da primeira cobrança. Você pode selecionar **Sem teste grátis** para não oferecê-lo, ou pode oferecer um período de teste gratuito de **7 dias**, **14 dias** ou **Personalizado**, onde você deve colocar a quantidade de dias que desejar.

> * **Duração da Assinatura:** você pode definir se seu plano de assinatura estará ativo por tempo ilimitado ou se acabará automaticamente após uma certa quantidade de recorrências de cobrança. Selecione **Ilimitado** se quiser que ele permaneça ativo indefinidamente. Se quiser que termine automaticamente, selecione **Limitado** e defina quantas recorrências de cobrança você deseja que sejam feitas.

Reviewer: ```suggestion
* **Duração da assinatura:** você pode definir se seu plano de assinatura estará ativo por tempo ilimitado ou se acabará automaticamente após uma certa quantidade de recorrências de cobrança. Selecione **Ilimitado** se quiser que ele permaneça ativo indefinidamente. Se quiser que termine automaticamente, selecione **Limitado** e defina quantas recorrências de cobrança você deseja que sejam feitas.
```

> * **Data de Cobrança:** escolha o dia do mês em que a assinatura será cobrada. _Esta opção está disponível apenas para planos com frequência de cobrança mensal_.

Reviewer: ```suggestion
* **Data de cobrança:** escolha o dia do mês em que a assinatura será cobrada. _Esta opção está disponível apenas para planos com frequência de cobrança mensal_.
```

> * **Cobrar valor proporcional na data de adesão:** Se esta opção estiver ativada, os clientes pagarão um valor proporcional ao se inscreverem pelos dias restantes até a primeira cobrança mensal. Se desativada, o valor adicional não será cobrado.

Reviewer: ```suggestion
* **Cobrar valor proporcional na data de adesão:** se esta opção estiver ativada, os clientes pagarão um valor proporcional ao se inscreverem pelos dias restantes até a primeira cobrança mensal. Se desativada, o valor adicional não será cobrado.
```

> * **Código de Referência:** Permite adicionar um código de referência para ajudá-lo a identificar o plano, caso você tenha vários planos semelhantes ativos.

Reviewer: ```suggestion
* **Código de referência:** permite adicionar um código de referência para ajudá-lo a identificar o plano, caso você tenha vários planos semelhantes ativos.
```

> * **Para onde você gostaria de redirecionar seu cliente?:** Você pode adicionar uma URL para a qual o cliente será redirecionado uma vez que a assinatura for concluída, como por exemplo seu site.

Reviewer: ```suggestion
* **Para onde você gostaria de redirecionar seu cliente?:** adicione uma URL para a qual o cliente será redirecionado uma vez que a assinatura for concluída, como o seu site, por exemplo.
```
